internal/acceptance/acceptance.go

```diff
@@ -62,6 +62,7 @@ func getFromEnvDefault(varName string, defaultValue string) string {
 	return defaultValue
 }
 
+// TODO: seems like this is sweeper config and broadly used; why is it non-standard?
 func GetConfigForNonStandardMetalTest() (*config.Config, error) {
```

Comment:
Doesn't necessarily need to happen in this PR, but IMO this should be `GetConfigForSweeper` and should fully replicate the behavior of the analogous `sharedConfigForRegion` in the `equinix` package